pages/common/stl2gts.md

```diff
@@ -0,0 +1,24 @@
+# stl2gts
+
+> Convert STL files to GTS (GNU triangulated surface library) format.
```

Comment:
```suggestion
> Convert STL files to GTS (GNU triangulated surface library) file format.
```

Reply:
I think `GTS format` is legit. It's also what is used in the man page.

Reply:
It might be good to also add "the" before "GTS."

Reply:
While that's technically correct, it sonuds really bad. Are you sure?

Reply:
> It might be good to also add "the" before "GTS."

I disagree, it makes GTS the noun, rather than adjective

Reply:
Something about the current wording feels like it's missing an article or preposition.  Maybe "Convert STL files into the GTS ..."?

Reply:
I know I said I disagree, yes, after reading a few more times over, it makes more sense

Reply:
I neither like `the GTS`, nor `GTS file format`, but in combination they are fine.

Reply:
Resolving since applied.